pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -266,19 +266,19 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
       split = ",",
       arity = "1..*",
       converter = RpcApisConverter.class,
-      description = "Comma separated APIs to enable on JSON-RPC channel. default: ${DEFAULT-VALUE}")
+      description = "Comma separated APIs to enable on HTTP JSON-RPC channel (default: ${DEFAULT-VALUE})")
   private final Collection<RpcApi> rpcHttpApis = DEFAULT_JSON_RPC_APIS;
 
   @Option(
       names = {"--rpc-http-authentication-enabled"},
       description =
-          "Set if the JSON-RPC service should require authentication (default: ${DEFAULT-VALUE})")
+          "Set if the RPC HTTP service should require authentication (default: ${DEFAULT-VALUE})")
```

Comment:
Shouldn't this be `Set if the JSON-RPC HTTP service should ...`

Reply:
can we remove the "should"?